tests/test_app/Plugin/BarPlugin/config/Migrations/20200208100000_bar_migration.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright Copyright (c) 2020 Juan Pablo Ramirez and Nicolas Masson
```

Comment:
We'll also need to add the CakePHP copyright to these files int addition to this copyright notice.